pkg/controller/controller.go

```diff
@@ -230,8 +230,7 @@ func (c *Controller) initRoleBinding() {
 	default:
 		c.PodServiceAccountRoleBinding = obj.(*rbacv1beta1.RoleBinding)
 		c.PodServiceAccountRoleBinding.Namespace = ""
-		c.PodServiceAccountRoleBinding.ObjectMeta.Name = c.PodServiceAccount.Name
-		c.PodServiceAccountRoleBinding.RoleRef.Name = c.PodServiceAccount.Name
+		c.PodServiceAccountRoleBinding.ObjectMeta.Name = fmt.Sprintf("%s:%s", c.PodServiceAccount.Name, "zalando-postgres-operator")
```

Comment:
Not sure if this will work with helm chart. Could you instead use `fmt.Sprintf` for the hard-coded `PodServiceAccountRoleBindingDefinition` and use `c.PodServiceAccount.Name` for `metadata.name` and `roleRef.name`? And then remove these two lines here.

Reply:
So use fmt.Sprintf in the default, remove these lines, and then define a custom rolebindingdefinition? That would also work, I guess, and be somewhat cleaner.

Reply:
yes. and if you don't define the rolebinding, clusterrole is the same as service account as it is right now